### PR TITLE
Update Russian Translation + a few typos

### PR DESCRIPTION
**Pull request by lufog**

### CODE REVIEW

File: SandboxiePlus/SandMan/sandman_ru.ts [hunk 1]
```diff
@@ -1675,369 +1697,390 @@ Note: The update check is often behind the latest GitHub release to ensure that
...
-        <location filename="SandMan.cpp" line="1616"/>
+        <location filename="SandMan.cpp" line="1995"/>
         <source>In the Plus UI this functionality has been integrated into the main sandbox list view.</source>
-        <translation type="unfinished"></translation>
+        <translation>В интерфейсе Plus эта функция была интегрирована в основное представление списка песочницы.</translation>
     </message>
     <message>
-        <location filename="SandMan.cpp" line="1617"/>
+        <location filename="SandMan.cpp" line="1996"/>
         <source>Using the box/group context menu you can move boxes and groups to other groups. You can also use drag an drop to move the items around. Alternatively you can also use the arow keys while holding ALT down to move items up and down within thair group.&lt;br /&gt;You can create new boxes and groups from the Sandbox menu.</source>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_ru.ts:1894]: - Typos to fix in the source file: https://github.com/sandboxie-plus/Sandboxie/blob/218797f14b1a6cda66d9cd07d29fb13534bfaa07/SandboxiePlus/SandMan/SandMan.cpp#L1996-L1997

e.g.
`Using the box/group context menu, you can move boxes and groups to other groups. You can also use drag and drop to move the items around. " `
`"Alternatively, you can also use the arrow keys while holding ALT down to move items up and down within their group.<br />" `

File: SandboxiePlus/SandMan/sandman_ru.ts [hunk 2]
```diff
@@ -1675,369 +1697,390 @@ Note: The update check is often behind the latest GitHub release to ensure that
...
     <message>
-        <location filename="SandMan.cpp" line="1470"/>
+        <location filename="SandMan.cpp" line="1545"/>
         <source>&lt;br /&gt;&lt;a href=&quot;https://sandboxie-plus.com/go.php?to=sbie-get-cert&quot;&gt;Become a project supporter&lt;/a&gt;, and receive a &lt;a href=&quot;https://sandboxie-plus.com/go.php?to=sbie-cert&quot;&gt;supporter certificate&lt;/a&gt;</source>
         <translation>&lt;br /&gt;&lt;a href=&quot;https://sandboxie-plus.com/go.php?to=sbie-get-cert&quot;&gt;Стань сторонником проекта&lt;/a&gt;, и получи &lt;a href=&quot;https://sandboxie-plus.com/go.php?to=sbie-cert&quot;&gt;сертификат сторонника&lt;/a&gt;</translation>
     </message>
     <message>
-        <location filename="SandMan.cpp" line="1616"/>
+        <location filename="SandMan.cpp" line="1995"/>
         <source>In the Plus UI this functionality has been integrated into the main sandbox list view.</source>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_ru.ts:1889]: - Missing comma after `In the Plus UI`
https://github.com/sandboxie-plus/Sandboxie/blob/ce20065242071c496f7277c4b3e69e2bd16e0cf9/SandboxiePlus/SandMan/SandMan.cpp#L1998

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by lufog** [comment]
@isaak654, done. But I don't know why the checks fail.

**Comment by isaak654** [comment]
> done. But I don't know why the checks fail.

One of the changes committed in https://github.com/sandboxie-plus/Sandboxie/commit/b563bab93d183528cef01a85a9d98414a7804046

**Pull request merged**